components/src/md3/Hero/Hero.tsx

```diff
@@ -10,7 +10,7 @@ import { useFontScale } from '../__contexts__/font-scale-context';
 type HeroStyles = {
     root?: ViewStyle;
     iconWrapper?: ViewStyle;
-    values?: ViewStyle;
+    channelValues?: ViewStyle;
```

Comment:
If you changed it here, you will want to update the readmes. And mention it in the change log, because this is a breaking change.

Reply:
reverting back to values
as it would create unnecessary changes to already used code 
and add further steps in migration to new component library